diff --git a/BUILD.bazel b/BUILD.bazel
index 6b0bed0..77a6757 100644
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -1,6 +1,6 @@
-load("@build_bazel_rules_apple//apple:ios.bzl", "ios_application", "ios_unit_test")
-load("@build_bazel_rules_apple//apple:macos.bzl", "macos_unit_test")
-load("@build_bazel_rules_swift//swift:swift.bzl", "swift_interop_hint", "swift_library")
+load("@rules_apple//apple:ios.bzl", "ios_application", "ios_unit_test")
+load("@rules_apple//apple:macos.bzl", "macos_unit_test")
+load("@rules_swift//swift:swift.bzl", "swift_interop_hint", "swift_library")
 
 package(default_visibility = ["//visibility:public"])
 
diff --git a/MODULE.bazel b/MODULE.bazel
index 2d092ab..b0b3ce3 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -1,8 +1,8 @@
 module(name = "promises", version = "2.4.0")
 
 bazel_dep(name = "rules_cc", version = "0.1.1")
-bazel_dep(name = "rules_apple", version = "3.20.1", repo_name = "build_bazel_rules_apple")
-bazel_dep(name = "rules_swift", version = "2.7.0", repo_name = "build_bazel_rules_swift")
+bazel_dep(name = "rules_apple", version = "3.20.1")
+bazel_dep(name = "rules_swift", version = "2.7.0")
 bazel_dep(name = "platforms", version = "0.0.11")
 bazel_dep(name = "apple_support", version = "1.20.0")

--
2.49.0
